cachito/web/content_manifest.py

```diff
@@ -153,6 +156,24 @@ def _process_standard_package(self, pkg_type, package, dependency):
         if not dependency.dev:
             pkg_type_data[package]["dependencies"].append(icm_dependency)
 
+    def process_rubygems_package(self, package, dependency):
+        """
+        Process RubyGems package.
+
+        :param Package package: the RubyGems package to process
+        :param Dependency dependency: the RubyGems package dependency to process
+        """
+        if dependency.type == "rubygems":
+            parent_package_name = self.request.repo.removesuffix(".git").split("/")[-1]
```

Comment:
It might be good to re-use the logic from scm.py for this https://github.com/containerbuildsystem/cachito/blob/35fb8db4a8146422a8772f4fb5b261c2cbaef2b2/cachito/workers/scm.py#L300

That would avoid a corner case where e.g. `https://github.com/containerbuildsystem/cachito/` resolves the package name to `''`

`repo_name` would return `containerbuildsystem/cachito`, so you might still want to split it and take only the actual repo name

Note that importing `cachito.workers.scm` in the `cachito.web.*` code probably won't work (missing git), so you would need to move the repo name logic somewhere else, probably into `cachito.common.utils`

Reply:
Hmm, but `repo_name` has a different corner case:
```python
>>> repo_name("git@github.com:containerbuildsystem/cachito.git")
'git@github.com:containerbuildsystem/cachito'
```
But either way, it would be good to use the same repo name logic here and in #692 and address the trailing slash corner case

Reply:
I both cases, we're getting URL from the request, so I don't think we need to care about PURL format conversion.

Reply:
Ah, I didn't mean PURL conversion, I meant SSH urls. Cachito does support those, and some requests actually use them:
```json
{
  "ref": "f7f69d8762afe9a936d1a6a26d4d06444e8b8929",
  "repo": "git@github.com:openshift/openshift-velero-plugin.git"
  ...
}
```

Reply:
But since you split on `/`  and take the last elem, that's not actually a problem :+1: